src/tests/test_parse.py

```diff
@@ -297,28 +298,42 @@ def complete_with_defaults(mapping):
 
 
 @pytest.mark.parametrize(
-    "line, expected",
+    "line, cmd_from, expected_error_message",
     [
         (
-            "some-argument",
-            {"line": ["some-argument"]},
-        ),
-        (
-            "a b c",
-            {"line": ["a", "b", "c"]},
+            "duckdb:// --alias test1 --alias test2",
+            "sql",
+            (
+                "Duplicate arguments in %sql. \
+Please use only one of each of the following: --alias"
+            ),
         ),
+    ],
+)
+def test_magic_args_raises_UsageError(ip, line, cmd_from, expected_error_message):
```

Comment:
do not use capital letters in test names

Reply:
[Fix1](https://github.com/ploomber/jupysql/pull/908/files#diff-6693dbe195b58d279ef47166dd1c83c4fe8153dd95a1c1521330f0e2d60033c2R172)
[Fix2](https://github.com/ploomber/jupysql/pull/908/files#diff-edd543175e806ac6654196270596c90240cc3a3cfdcba20d35d997207e4555f0R314)